### PR TITLE
docs(paragraph): added documentation to the "alignment" method

### PR DESCRIPTION
**Pull request by DreadedHippy**
Added documentation to the `alignment` method of the `Paragraph` widget.

This is really just 1 commit, the second commit was made because the first wasn't signed

### CODE REVIEW

File: src/widgets/paragraph.rs [hunk 1]
```diff
@@ -196,6 +196,18 @@ impl<'a> Paragraph<'a> {
...
 
+    /// Set the text alignment for the given paragraph
+    /// 
+    /// The alignment is a variant of the [`Alignment`] enum which can be one of Left,
+    /// Right, or Center.
+    /// 
+    /// ```rust
+    /// # use ratatui::prelude::*;
+    /// # use ratatui::widgets::{Paragraph, Wrap};
+    /// # use ratatui::layout::Alignment;
```
Comment (joshka) [inline src/widgets/paragraph.rs:207]: This is included in the `prelude::*` import, so isn't necessary.
```suggestion
```

File: src/widgets/paragraph.rs [hunk 2]
```diff
@@ -196,6 +196,18 @@ impl<'a> Paragraph<'a> {
         self
     }
 
+    /// Set the text alignment for the given paragraph
+    ///
```
Comment (joshka) [inline src/widgets/paragraph.rs:200]: There's a extra space at the end of this line and the other empty line below that rustfmt will delete.
```suggestion
    ///
```
run the formatter locally using:
```shell
cargo +nightly fmt --all
```

File: src/widgets/paragraph.rs [hunk 3]
```diff
@@ -196,6 +196,18 @@ impl<'a> Paragraph<'a> {
         self
     }
 
+    /// Set the text alignment for the given paragraph
+    /// 
+    /// The alignment is a variant of the [`Alignment`] enum which can be one of Left,
+    /// Right, or Center.
+    ///
```
Comment (joshka) [inline src/widgets/paragraph.rs:203]:
```suggestion
    ///
```

**Review by joshka — CHANGES_REQUESTED** [review]
To fix this up rather than creating a new PR, you can rebase and squash the commits:
```
git rebase -i
change the second "pick" to "fixup" (or just "f"), save and close
git push origin main
```

**Comment by DreadedHippy** [comment]
I've resolved all suggested changes.

Although the following error kept popping up:
```

error[E0432]: unresolved import `sys`
  --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\termion-2.0.1\src\raw.rs:29:5
   |
29 | use sys::Termios;
   |     ^^^ maybe a missing crate `sys`?
   |
   = help: consider adding `extern crate sys` to use the `sys` crate
```

I pushed with the '--no-verify' flag

**Review by joshka — APPROVED** [review]

**Comment by joshka** [comment]
Apologies - I just accidentally pushed the main branch over your main branch rather than my local copy of your main branch rebased to your main branch (which closes the PR and locks me out of the branch), so this shows as closed. I've merged it out of band to fix things up however).

Thanks for your first contribution to ratatui!

**Comment by DreadedHippy** [comment]
Ayyyyy! 🚀🚀🚀🚀🚀

**Pull request closed without merging**